UI/EncyclopediaDetailPanel.cpp

```diff
@@ -408,15 +442,57 @@ namespace {
             //for (auto str : GetStringTable().
 
         } else {
-            // list categories
-            std::map<std::string, std::vector<EncyclopediaArticle> >::const_iterator category_it =
-                encyclopedia.articles.find(dir_name);
-            if (category_it != encyclopedia.articles.end()) {
-                for (const EncyclopediaArticle& article : category_it->second) {
-                    sorted_entries_list.insert(std::make_pair(UserString(article.name),
-                        std::make_pair(LinkTaggedText(TextLinker::ENCYCLOPEDIA_TAG, article.name) + "\n",
-                                       article.name)));
-                }
+            // Add articles for content tagged definitions
```

Comment:
explain what "content tagged definitions" are a bit? lacking a lot of context, this is hard to understand.